src/OrbitGl/CoreMathTest.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include "CoreMath.h"
+
+namespace orbit_gl {
+
+ClosedInterval kFakeClosedInterval{0, 3};
+
+TEST(ClosedInterval, FromValues) {
+  EXPECT_EQ(kFakeClosedInterval, ClosedInterval::FromValues(0, 3));
+  EXPECT_EQ(kFakeClosedInterval, ClosedInterval::FromValues(3, 0));
+}
```

Comment:
This makes not a lot of sense to me. If you don't force your users to use `FromValues` it's better not to have it IMHO.

Reply:
Yes, I thought about that. I will revert it.